cpp/ql/lib/semmlecode.cpp.dbscheme.stats

```diff
@@ -78,7 +78,7 @@
         </e>
         <e>
             <k>@using_enum_declaration</k>
-            <v>1</v>
+            <v>0</v>
```

Comment:
Looks like you didn't run the internal workflow we have to generate additional data from out tests.

Reply:
I ran the generate-db-stats workflow, downloaded the artefact, and merge was apparently successful:

```
[13:53:32] [DCA info] Normalizing stats by user-provided 'exprs'.
[13:53:32] [DCA info] Using stats from tests '~/Downloads/testsuite_stats.xml'.
[13:53:32] [DCA info] Using static stat files from /Users/calum/projects/codeql-dca/static-stats/cpp.
```

what am I missing?

Reply:
That's not possible for me to tell from this alone. Let's discuss internally?